pages/common/tmux.md

```diff
@@ -9,28 +9,28 @@
 
 - Start a new named session:
 
-`tmux new-session -s {{name}}`
+`tmux new -s {{name}}`
 
 - List existing sessions:
 
 `tmux ls`
 
 - Attach to the most recently used session:
 
-`tmux attach-session`
+`tmux attach`
```

Comment:
Maybe this example could be one for attaching to a named session instead? I don't see that mentioned here, and it would be nice to include that.

Reply:
My thought process was that, after you attach to the most recent session, you can change the currently selected session with the following controls. Anyway, to attach to a named session, one would do `tmux attach -t {{name}}`. I can put this one instead, as you prefer.

Reply:
Oh I see! I didn't notice that. I don't use tmux myself, so I apologise. Let's leave this as-is.